sgx-tools/gen-quote.go

```diff
@@ -66,18 +63,31 @@ For example, generate the quote file according to the given local report file:
 			return fmt.Errorf("Unsupport quote type: %v", quoteType)
 		}
 
+		var attester attestation.Attester
+
 		if strings.EqualFold(quoteType, intelsgx.QuoteTypeEpidUnlinkable) || strings.EqualFold(quoteType, intelsgx.QuoteTypeEpidLinkable) {
 			spid := context.String("spid")
 			if spid == "" {
 				return fmt.Errorf("spid can't be empty in both epid for unlinkable and epid for linkable modes")
 			}
 
-			if len(spid) != spidLength*2 {
-				return fmt.Errorf("Spid must be %d-character long", spidLength*2)
+			if len(spid) != sgx.SpidLength*2 {
+				return fmt.Errorf("Spid must be %d-character long", sgx.SpidLength*2)
+			}
+
+			p := parseSgxEpidAttester(quoteType, spid)
+			attester, err = attestation.NewAttester(sgx.AttestationEpid, p)
+			if err != nil {
+				return err
+			}
+		} else {
+			attester, err = attestation.NewAttester(sgx.AttestationEcdsa, nil)
+			if err != nil {
+				return err
```

Comment:
Change to use this code style to reduce redundancy:
```go
var p <type> = nil

if ..
    p = ....
else 
    ...

attester, err = attestation.NewAttester(sgx.AttestationEpid, p)
```

Reply:
> Change to use this code style to reduce redundancy:
> 
> ```go
> var p <type> = nil
> 
> if ..
>     p = ....
> else 
>     ...
> 
> attester, err = attestation.NewAttester(sgx.AttestationEpid, p)
> ```

Done, thanks.

Reply:
> Change to use this code style to reduce redundancy:
> 
> ```go
> var p <type> = nil
> 
> if ..
>     p = ....
> else 
>     ...
> 
> attester, err = attestation.NewAttester(sgx.AttestationEpid, p)
> ```

Done, thanks.